c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -1,7 +1,20 @@
-VENV_BIN ?= .build/venv/bin
 PYTHON_VERSION = $(shell $(VENV_BIN)/python -c "import sys; print('%i.%i' % (sys.version_info.major, sys.version_info.minor))" 2> /dev/null)
 PACKAGE = {{package}}
+OPERATING_SYSTEM ?= LINUX
+
+ifeq ($(OPERATING_SYSTEM), WINDOWS)
+VENV_BIN = .build/venv/Scripts
+PYTHON_BIN_POSTFIX = .exe
+NODE_BIN_POSTFIX = .cmd
+FIND ?=$(CYGWIN_PATH)/bin/find.exe
```

Comment:
no default value for CYGWIN_PATH ?
at least it will need to be documented to say it's absolutly mandatory